bobtemplates/plone/theme_barceloneta/DEVELOP_THEME.rst.bob

```diff
@@ -0,0 +1,42 @@
+Theme development
+-----------------
+
+In order to develop the theme you need to install grunt-cli global::
+
+    $ sudo npm install -g grunt-cli
+
+You can use the helper script `npm-install`::
+
+    $ ./bin/npm-install
+
+It will run npm install in package root and in the theme folder.
+
+After executing these commands you can run grunt on the package root folder to watch for any less changes::
+
+    $ grunt watch
+
+This will make sure that the many .less files are compiled to .css on the fly and then served up from the theme.
+
+You can also use this command to automaticly reload the browser after changes::
+
+    $ grunt plone-bsync
+
+It will run a Grunt browser sync task in a new browser window and will reload after every change of the less files in the less folder. If your Plone is not running on port 8080 you have to adjust the proxy option in the Gruntfile.js.
+
+If you prefer to do a one time compile of the less files you can run::
+
+    $ grunt compile
+
+or use the helper script from buildout::
```

Comment:
Please check this, i don't think that it's working so far, because you don't update the existing buildout config, which is coming from the addon template.